.gitignore

```diff
@@ -1 +1,2 @@
 schemes
+*.zwc
```

Comment:
Can you add a comment her mentioning what this is.

Reply:
sure! It is to ignore the precompiled zsh files my ZSH plugin manager `zcomet` (and others) creates in-place to improve load time:
```
$ ls -l base16-shell.plugin.zsh*
-rw-rw-r-- 1 josh josh  214 Jul 13 10:41 base16-shell.plugin.zsh
-r--r--r-- 1 josh josh 1000 Jul  4 12:47 base16-shell.plugin.zsh.zwc
```

Reply:
that's a really good point - there could be some interesting stuff in there :)
I'm adding it now!